docs/app/partials/layout-introduction.tmpl.html

```diff
@@ -0,0 +1,364 @@
+      </tr>
+      </thead>
+      <tr>
+        <td>layout</td>
+        <td>flex</td>
+        <td>Sets default layout direction &amp; flex unless overridden by another breakpoint.</td>
+      </tr>
+      <tr>
+        <td>layout-xs</td>
+        <td>flex-xs</td>
+        <td>width &lt; <b>600</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-gt-xs</td>
+        <td>flex-gt-xs</td>
+        <td>width &gt;= <b>600</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-sm</td>
+        <td>flex-sm</td>
+        <td><b>600</b>px &lt;= width &lt; <b>960</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-gt-sm</td>
+        <td>flex-gt-sm</td>
+        <td>width &gt;= <b>960</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-md</td>
+        <td>flex-md</td>
+        <td><b>960</b>px &lt;= width &lt; <b>1280</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-gt-md</td>
+        <td>flex-gt-md</td>
+        <td>width &gt;= <b>1280</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-lg</td>
+        <td>flex-lg</td>
+        <td><b>1280</b>px &lt;= width &lt; <b>1920</b>px</td>
+      </tr>
+      <tr>
+        <td>layout-gt-lg</td>
+        <td>flex-gt-lg</td>
+        <td>width &gt;= <b>1920/b>px</td>
+      </tr>
+      <tr>
+        <td>layout-xl</td>
+        <td>flex-xl</td>
+        <td>width &gt;= <b>1920</b>px</td>
+      </tr>
+    </table>
+
+  <p>Below is an example usage of the Responsive Layout API:</p>
+
+  <hljs lang="html">
+    <div layout='column' class="zero">
+
+      <div flex="33" flex-md="{{ vm.box1Width }}" class="one"></div>
+      <div flex="33" layout="{{ vm.direction }}" layout-md="row" class="two">
+
+        <div flex="22" flex-md="10" hide-lg class="two_one"></div>
+        <div flex="30px" show hide-md="{{ vm.hideBox }}" flex-md="25" class="two_two"></div>
+        <div flex="20" flex-md="65" class="two_three"></div>
+
+      </div>
+      <div flex class="three"></div>
+
+    </div>
+  </hljs>
+
+  <br/>
+
+  <p>
+  This Layout API means that developers no longer have to worry about the details of FlexBox styling.
+  The developer uses the Layout HTML API to specify <b><i>intention</i></b> and the Layout engine handles all the issues of CSS flexbox styling.
+  </p>
+
+  <p class="layout_note">
+    The Layout engine will even report console warnings for Layout conflicts and issues that are not automatically resolved.
+  </p>
+
+
+  <br/><br/>
+  <hr>
+  <br/>
+
+  <h3>Under-the-Hood</h3>
+
+  <p>
+    Due to performance problems when using Attribute Selectors with <b>IE10</b> and <b>IE11</b> browsers, our attributes are
+    dynamically converted to class selectors at runtime. And the Layout CSS classNames and styles have been each been
+    predefined in the <code>angular-material.css</code> stylesheet.
+  </p>
+
+  <p class="layout_note">
+    This means the HTML design-time markup is not affected. Developers should continue to use Layout attribute selectors in the HTML
```

Comment:
I think this paragraph is confusing. I don't know what "design-time" means or what it's not affected by. "Developers should continue to use Layout attribute": as opposed to what?

I would replace this paragraph with a note that the Layout directives do more than directly apply CSS classes, and that the classes may change between releases. Using the directives is the only supported API.